test/integration/integration-test.js

```diff
@@ -80,8 +86,16 @@ function teardown(done) {
 describe.skip('integration tests', function() {
   const instructions = {maxLedgerVersionOffset: 100};
   this.timeout(TIMEOUT);
+  const transactions = [];
 
+  // beforeEach(setup);
   beforeEach(setup);
+  beforeEach(function() {
+    if (!this.startLedgerVersion) {
```

Comment:
Yeah, you're right, it has `before` method, but for `startLedgerVersion` I need value from first `this.api.connect` run, so no other way.